common/src/sql/dbinit.sql

```diff
@@ -148,7 +148,28 @@ CREATE TABLE omicron.public.Region (
  * Allow all regions belonging to a disk to be accessed quickly.
  */
 CREATE INDEX on omicron.public.Region (
-    disk_id
+    volume_id
+);
+
+/*
+ * A volume within Crucible
+ */
+CREATE TABLE omicron.public.volume (
+    id UUID PRIMARY KEY,
+    time_created TIMESTAMPTZ NOT NULL,
+    time_modified TIMESTAMPTZ NOT NULL,
+    time_deleted TIMESTAMPTZ,
+
+    /* child resource generation number, per RFD 192 */
+    rcgen INT NOT NULL,
+
+    /*
+     * A JSON document describing the construction of the volume, including all
+     * sub volumes. This is what will be POSTed to propolis, and eventually
+     * consumed by some Upstairs code to perform the volume creation. The Rust
+     * type of this column should be Crucible::VolumeConstructionRequest.
+     */
+    data TEXT NOT NULL /* JSONB */
```

Comment:
the above comment is still correct, but I don't think we should keep the "JSONB" comment here unless the type of this column changes to that